discovery/manager/manager.py

```diff
@@ -39,6 +39,8 @@ def get_ansible_vars_from_input_context(input_context: InputContext) -> dict:
         vars['ansible_ssh_extra_args'] = input_context.ansible_ssh_extra_args
         vars['ansible_python_interpreter'] = input_context.ansible_python_interpreter
         vars['ansible_ssh_private_key_file'] = input_context.ansible_ssh_private_key_file
+        vars['ansible_common_remote_group'] = input_context.ansible_common_remote_group
+        vars['ansible_become_password'] = input_context.ansible_become_password
```

Comment:
are we also adding password flags? As of now, discovery expects password less ssh configuration via keys. Are we adding this enhancement?

Reply:
Yea, these are Ansible capabilities to connect to host. We are just proxies all possible parameters to Ansible runner.